Source/Core/DolphinWX/Main.cpp

```diff
@@ -208,9 +208,6 @@ void DolphinApp::MacOpenFile(const wxString& fileName)
 
 void DolphinApp::AfterInit()
 {
-  if (!m_batch_mode)
-    main_frame->UpdateGameList();
```

Comment:
> The GameListCtrl posts the message to itself when it's created, the main app object doesn't need to do it.

Alright, so this particular piece of code isn't the problem.

> The reason I didn't implement full rescan on init is because redrawing the list takes a noticable amount of time, and if the files have not changed from the cached version, this redraw is annoying. It is simple enough to detect that and skip the redraw, though. I could do that in another PR.

Sounds good.